zephyr/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2022 Legrand North America, LLC.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
+
+
+# Detect the platform reliably
+if(ZEPHYR_BASE)
```

Comment:
Why not `if(NOT ZEPHYR_BASE)`

Reply:
No particular reason.  What is your reasoning for preferring one over the other?